db/models.py

```diff
@@ -1 +1,26 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
```

Comment:
You should add choices here

Reply:
Sorry for telling you to add `choices` here, I did as you did, but I got a comment about it, but it turns out it's not necessary)